### ISSUE TITLE
ValueError: list.remove(x): x not in list with sismic 1.4

### ISSUE THREAD

**Issue by naydi-nebras** [acsone/scobidoo#15]
```
2018-10-22 12:55:15,290 26553 ERROR nacsu11 odoo.http: Exception during JSON request handling.
Traceback (most recent call last):
  File "/opt/odoo/v11ce/server/odoo/http.py", line 650, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/v11ce/server/odoo/http.py", line 310, in _handle_exception
ValueError: list.remove(x): x not in list
```

**Comment by sbidoul** [comment]
Hi, I could reproduce this with sismic 1.4. With sismic 1.3, the problem does not occur.

I'll debug this, in the meantime, stick to sismic 1.3.

**Comment by sbidoul** [comment]
Thanks for reporting. This is resolved with sismic 1.4.1 (see https://github.com/AlexandreDecan/sismic/issues/89)
